core/src/main/java/com/redhat/lightblue/client/request/LightblueHealthRequest.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.lightblue.client.request;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.redhat.lightblue.client.http.HttpMethod;
+
+public class LightblueHealthRequest extends LightblueRequest{
```

Comment:
Is this intended to be for /health or /diagnostics?

These are two distinct APIs with different response schemas. The (future) intention of /health is to return an empty response with just a status code 200 or 503. We may add a response body here in the future but the primary use case is a boolean "should you send me requests?" answer.

The /diagnostics endpoint is different, it's a collection of various health checks and details of their results and potentially other useful information. This has a stable schema.

I would have support for both, though right now we only have /diagnostics implemented.